maps/nerva/nerva_overmap.dm

```diff
@@ -65,3 +65,9 @@
 	shuttle = "Hadrian"
 	fore_dir = EAST
 	vessel_mass = 1000
+
+/obj/effect/overmap/visitable/ship/combat/nerva/pve_mapfire(projectile_type)
+	if(ispath(projectile_type))
+		var/turf/start_turf = spaceDebrisStartLoc(pick(GLOB.cardinal), src.z)
+		var/turf/target_turf = locate(100, 100, src.z) //set up values for enemy ships
+		launch_atom(projectile_type, start_turf, target_turf)
```

Comment:
If things remain how they are, won't the Nerva be shooting at itself here?

Reply:
Actually it'll be shooting at the actual overmap :L

Reply:
Welp. No idea why github was showing me old files.. GG. I see you already fixed this